CodeForPoznan/codeforpoznan.pl_v3
# [164] Refactor title classes

Styles (mostly title classes) were refactored.

**Testing**:
1. Fetch this PR: `git fetch origin pull/211/head:<local-branch-name>`
2. Run app: `make start`
3. Go to -> `http://localhost:8080/`
4. Check if all sections titles have underscore.
5.  Check if on white background are blue titles and conversely.
